docs/pages/2020_BIGMAP/sections/browse.rst

```diff
@@ -0,0 +1,66 @@
+.. _BIGMAP_2020_browse:
+
+Browsing your database
+======================
+
+This final section of the tutorial is optional, and has the purpose of introducing a couple of ways in which you can browse an AiiDA database.
+
+There are two main reasons you may want to browse a database.
+The first one is to get a general idea of the content of it, for which the explore tool of the `Materials Cloud <https://www.materialscloud.org/explore/menu>`_ can be of great use.
+The second one is a more specific search with the purpose of gathering more particular and well defined results.
+For this second use, we will reccommend direct queries via the ``QueryBuilder``.
+
+.. note::
+
+  Although we introduce them for different purposes, both of these tools can ultimately be used for general exploration or more specific searches.
```

Comment:
I'm not sure if you can use the `QueryBuilder` for exploration of the database. You already need to have a pretty good idea of how your data is structured, no?

Reply:
Well, this is the thing, I agree it is not ideal (and say it in the text), but *technically* the explore section **is** an interface for the querybuilder (the information shown is acquired via sending queries to the exposed database) so you can do all you do there in the python shell.

Reply:
True, all good 👍 , just a detail.